.github/workflows/notify-slack.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+name: 'Notify slack'
+
+on:
+  workflow_call:
+    inputs:
+      application:
+        description: 'The friendly name for the application'
+        type: string
+        required: true
+
+      environment:
+        description: |
+          The environment variables to pass to the render.  Each environment variable should be in format VAR=VAL with one on each line.
+          Example:
+            environment: |
+              FOO=bar
+              BAR=baz
+        type: string
+        required: false
+        default: ''
+
+      template:
+        description: 'The path to the template file to render'
+        type: string
+        required: true
```

Comment:
This should default to `${{ github.action_path }}/message.json.j2`, right?  Not required.